### PR TITLE
docs for keyset generation on anytrust orbits

### PR DESCRIPTION
**Pull request by GreatSoshiant**
In this part of the doc we show how an anytrust chain owner can generate keyset and set it up for their chains

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (Tristan-Wilson) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:12]: The minimum number of trusted committee members is configurable by the "assumed-honest" parameter in the keyset.

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (Tristan-Wilson) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:16]: (style comment, feel free to ignore)
"cognizant", "encompassing", "facilitate", "delineate", "empower" 
I'd make the the language here simpler and more direct/specific.

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (Tristan-Wilson) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:24]: assumed-honest is H in the formula below, so it's true that it (indirectly) sets the number of committee members that need to respond, but it might be misinterpreted here that it directly sets it.

K = (N + 1) - H

K: Number of committee members that must sign for Store to be considered successful 
N: Total number of committee members
H: Minimum number of honest members

**Review by Tristan-Wilson — COMMENTED** [review]
Comments from me are mainly asking for a bit more clarity around the assumed-honest parameter.

**Comment by symbolpunk** [comment]
In review.

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:10]:
```suggestion
**AnyTrust** chains rely on an external Data Availability Committee (DAC) to store data and provide it on demand, instead of using the parent chain as Data Availability (DA) layer. 
```

We use DAC elsewhere [[1]](https://docs.arbitrum.foundation/dao-glossary#data-availability-committee-dac); edit for consistency. This would also let us use quicklooks to expose term definitions (pulled from [Glossary CMS](https://www.notion.so/arbitrum/Glossary-CMS-0718756f84b44dcfa8907a72854550cf)) on hover.

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:12]:
```suggestion
The DAC has _N_ members; the AnyTrust protocol assumes that a minimum of _H_ DAC members maintain integrity. _H_ is the minimum number of trusted committee members on AnyTrust chains, configurable by the chain's owner via the `assumed-honest` parameter in the keyset. In scenarios where `K = (N + 1) - H` members of the DAC pledge to grant access to specific data, they must sign and attest they have the data for store to be considered successful.
```

Does this edit feel like an improvement?

> Consequently, in scenarios where `K = (N + 1) - H` members of the Committee pledge to grant access to specific data, and they must sign and attest they have the data for store to be considered successful.

I don't understand what we're trying to communicate here. Would you mind rearticulating / clarifying?

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:14]:
```suggestion
Each DAC member gets their own set of BLS public and private keys. It's important for every member to create their own new and secure BLS keys. They should do this on their own and make sure these keys are random and only for their use. If you need help generating BLS keys, check out our guide [here](https://docs.arbitrum.io/node-running/how-tos/running-a-daserver#generate-key).
```

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:16]:
```suggestion
The main blockchain (parent chain) needs to know the names and public keys of all DAC members in order to validate the integrity of data being batched and posted. A 'keyset' is a list of all DAC members' public keys. It also shows how many signatures are needed to approve a Data Availability Certificate. This design lets the chain owner modify the DAC's membership over time, and it lets DAC members change their keys if needed. See <a data-quicklook-from='inside-anytrust'>Inside Anytrust</a> for more information.
```

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:21]: @GreatSoshiant how do you feel about refactoring the procedural instructions out of this document, and into a dedicated how-to document?

Our [established conventions](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#2309f7f5db284126929065ac3c56fef8) distinguish between procedural and explanatory documents to make it easier for readers to find what they're looking for.

Not a blocker, but important if we want to maintain conventions (we can always address via another PR if this is time-sensitive).

If you opt to refactor now, Orbit docs auto-generates the sidebar from the directory structure, so creating a new how-to document in the `how-tos` directory should be all that's needed. We can also link to the how-to from the concept, and vice-versa, to improve discoverability of both.

**Review by symbolpunk — CHANGES_REQUESTED** [review]
Looks great! Pausing review to address a possible refactoring, will resume once decided.

File: arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md
Comment (GreatSoshiant) [reply arbitrum-docs/launch-orbit-chain/concepts/anytrust-orbit-chain-keyset-generation.md:21]: Can we stick to this type of doc for now @symbolpunk ? We can move to the how-to doc later if you agree.

**Review by symbolpunk — APPROVED** [review]
Marking as approved to unblock; will hold off on merging in case an SME would like to review one last time before publishing (cc @Tristan-Wilson)

**Review by Alirezaabbasi-hub — COMMENTED** [review]
Run

**Pull request merged**
